client/src/components/Visualization/charts/ZoomLine.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
+import {
+  VictoryAxis,
+  VictoryBrushContainer,
+  VictoryChart,
+  VictoryLine,
+  VictoryZoomContainer,
+} from 'victory';
+
+import { generateRandomHexColor } from '../../../utils/Utils';
+
+const MOCK_DATA = [
+  { a: new Date(1982, 1, 1), b: 125 },
+  { a: new Date(1987, 1, 1), b: 257 },
+  { a: new Date(1993, 1, 1), b: 345 },
+  { a: new Date(1997, 1, 1), b: 515 },
+  { a: new Date(2001, 1, 1), b: 132 },
+  { a: new Date(2005, 1, 1), b: 305 },
+  { a: new Date(2011, 1, 1), b: 270 },
+  { a: new Date(2015, 1, 1), b: 470 },
+];
+
+/*
```

Comment:
Minor nit but if this is `JSDoc` please format correctly. JSDoc is `/** bla bla */. use /* bla bla */ for block comments and implementation notes. 

https://google.github.io/styleguide/jsguide.html#jsdoc

Reply:
This is intended to be a block comment, not JSDoc!